### PR TITLE
doc: add Thread link metrics feature information

### PR DESCRIPTION
**Pull request by edmont**
Add a summarized description of the Link Metrics Thread 1.2 feature.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

KRKNWK-7918

### CODE REVIEW

File: doc/nrf/ug_thread_supported_features.rst [hunk 1]
```diff
@@ -45,3 +45,28 @@ Limitations for Thread 1.2 support
...
+
+The available metrics are:
+* Count of received PDUs.
+* Exponential Moving Average of the LQI.
+* Exponential Moving Average of the link margin (dB).
+* Exponential Moving Average of the RSSI (dBm).
+
+The main potential usage of this feature is to allow a battery powered device to lower its transmission power when the link quality with its parent is good enough.
+This way the device can save energy on each transmitted frame.
+The device interested in receiving link quality information is referred as Link Metrics Initiator, while the device delivering that information is referred as Link Metrics Subject.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_supported_features.rst:62]:
```suggestion
The device interested in receiving link quality information is referred to as Link Metrics Initiator, while the device delivering that information is referred to as Link Metrics Subject.
```

File: doc/nrf/ug_thread_supported_features.rst [hunk 2]
```diff
@@ -45,3 +45,28 @@ Limitations for Thread 1.2 support
...
+
+Developers can make use of the OpenThread API to obtain information about the link quality in three ways:
+* Single probe: a single shot MLE command that returns the requested metrics in the reply.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_supported_features.rst:65]:
```suggestion
* Single probe: a single-shot MLE command that returns the requested metrics in the reply.
```

File: doc/nrf/ug_thread_supported_features.rst [hunk 3]
```diff
@@ -45,3 +45,28 @@ Limitations for Thread 1.2 support
...
+* Forward Tracking Series: the Initiator configures the Subject to start tracking link quality information for every received frame.
+At any point, the Initiator retrieves the averaged values for that series by means of an MLE Data Request.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_supported_features.rst:67]:
```suggestion
  At any point, the Initiator retrieves the averaged values for that series by means of an MLE Data Request.
```
I believe this is part of the second bullet, so the indent is necessary.

File: doc/nrf/ug_thread_supported_features.rst [hunk 4]
```diff
@@ -45,3 +45,28 @@ Limitations for Thread 1.2 support
...
 
 * Due to code size limitation, the combination of complete set of Thread 1.2 features with the Bluetooth® LE multiprotocol support is not possible for the nRF52833 DKs.
+
+Link Metrics Probing Protocol
+=============================
+
+Thread 1.2 Specification introduces Link Metrics Probing Protocol as a mechanism to gather information about the link quality with neighbors.
+
+The available metrics are:
+* Count of received PDUs.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_supported_features.rst:55]:
```suggestion
The available metrics are:

* Count of received PDUs.
```
Empty line before a list so it renders properly.

File: doc/nrf/ug_thread_supported_features.rst (hunk 2 above)
Comment (wiba-nordic) [inline doc/nrf/ug_thread_supported_features.rst:65]:
```suggestion
Developers can make use of the OpenThread API to obtain information about the link quality in three ways:

* Single probe: a single shot MLE command that returns the requested metrics in the reply.
```
Another empty line here.

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
Need a couple of small changes, mostly so things will render correctly.

**Review by lmaciejonczyk — APPROVED** [review]

**Comment by edmont** [comment]
> Need a couple of small changes, mostly so things will render correctly.

Thanks, done.

**Review by wiba-nordic — APPROVED** [review]
Looks good!

**Pull request merged**